addon/components/materialize-tabs.js

```diff
@@ -0,0 +1,87 @@
+import Ember from 'ember';
+import layout from '../templates/components/materialize-tabs';
+
+export default Ember.Component.extend({
+  layout: layout,
+  tagName: 'ul',
+  tabWidth: 2,
+  classNames: ['tabs'],
+
+  tabComponents: [],
+  activeTab: null,
+
+  registerTab: function (tab) {
+    var tabs = this.get('tabComponents');
+    tabs.addObject(tab);
+  },
+
+  setIndicatorPosition: function (left, right, animateUpdate=true) {
```

Comment:
Handles moving the "active" indicator between tabs. Optionally, this can be done without animation (i.e., on initial indicator placement via `didInsertElement`)